Signum.React/Scripts/Lines/EntityBase.tsx

```diff
@@ -277,6 +314,19 @@ export class EntityBaseController<P extends EntityBaseProps> extends LineBaseCon
     );
   }
 
+  renderPasteButton(btn: boolean) {
+    if (this.props.ctx.readOnly || !this.props.type || (this.props.type.name != IsByAll && !this.props.type.isCollection))

```

Comment:
EntityBase should have a `paste` property which default value is set to `this.props.type.name == IsByAll` in `getDefaultProps`, similar to how the `create`/`find`/`view` properties works.

Reply:
done 👍